File: app/src/main/java/com/example/qiuchenly/comicparse/Bean/HotComicStrut.java
package com.example.qiuchenly.comicparse.Bean;

public class HotComicStrut {
    public String bookName;
    public String bookImgSrc;
    public String lastedPage_name;
    public String lastedPage_src;
    public String bookLink;
    public String author;

    @Override
    public String toString() {
        return bookName + "|" + bookImgSrc + "|" + lastedPage_name + "|" + lastedPage_src + "|" + bookLink;
    }
}

File: 漫画Main/src/main/java/com/example/qiuchenly/comicparse/Bean/HotComicStrut.kt
package com.example.qiuchenly.comicparse.Bean

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.RealmClass

@RealmClass
open class HotComicStrut: RealmObject() {
    @PrimaryKey
    var bookName: String? = null
    var bookImgSrc: String? = null
    var lastedPage_name: String? = null
    var lastedPage_src: String? = null
    var bookLink: String? = null
    var author: String? = null

    override fun toString(): String {
        return "$bookName|$bookImgSrc|$lastedPage_name|$lastedPage_src|$bookLink"
    }
}